test/evergreen/tsan_warnings.supp

```diff
@@ -0,0 +1,9 @@
+# Suppress the listed TSan warnings. Load this file with 
+# export TSAN_OPTIONS="suppressions=path/to/wiredtiger/test/evergreen/tsan_warnings.supp"
+# before running tests with TSan
+
+# FIXME-WT-XXXX TSan is raising warnings about mutexes that are non-intuitive. These will be investigated separately.
```

Comment:
I've split this work into later tickets because it's either a large amount of effort (mutexes), or I want the changes in their own commit to review potential perf impacts (mtx_rw). I'll create the follow on tickets if we agree on this approach